gormapplication/application.go

```diff
@@ -137,7 +137,7 @@ func (g *GormDB) PermissionService() permissionservice.PermissionService {
 }
 
 func (g *GormDB) RoleManagementService() roleservice.RoleManagementService {
-	return roleservice.NewRoleManagementService(g)
+	return roleservice.NewRoleManagementService(g, g)
```

Comment:
I think we just need to create a new interface which will extends both Repositories and Transaction interfaces and just pass GormDB once. It's for a separate PR though. Let's not do it in this PR.

Reply:
Agreed, while doing this - it looked a bit odd to me :)

Reply:
I am very likely changing this as part of #490